quickstarts/node-js/hapi/config.yml

```diff
@@ -1,6 +1,6 @@
 id: 2124da5b-0174-457a-be5a-e0068673b2b5
 name: hapi
-description: "## What is Hapi?\n\nLightweight and highly scalable backend server support built in Node.JS.\n\n## Get started!\n\nLeverage community expertise and instantly get value out of your telemetry data. This quickstart automatically instruments Hapi with the New Relic Node.js agent, and allows you to instantly monitor your Node application with out-of-the-box dashboards and alerts. Further leverage New Relic's APM capabilities by setting up [errors inbox](https://docs.newrelic.com/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/), [transaction tracing](https://docs.newrelic.com/docs/apm/transactions/transaction-traces/introduction-transaction-traces/), and [service maps](https://docs.newrelic.com/docs/understand-dependencies/understand-system-dependencies/service-maps/introduction-service-maps/).\n\n## More info\n\nCheck out the [documentation](https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs/) to learn more about New Relic monitoring for Hapi."
+description: "## What is Hapi?\n\nLightweight and highly scalable backend server support built in Node.JS.\n\n## Get started!\n\nLeverage community expertise and instantly get value out of your telemetry data. This quickstart automatically instruments Hapi with the New Relic Node.js agent, and allows you to instantly monitor your Node application with out-of-the-box dashboards and alerts. Further leverage New Relic's APM capabilities by setting up [errors inbox](https://docs.newrelic.com/docs/apm/apm-ui-pages/errors-inbox/errors-inbox/), [transaction tracing](https://docs.newrelic.com/docs/apm/transactions/transaction-traces/introduction-transaction-traces/), and [service maps](https://docs.newrelic.com/docs/understand-dependencies/understand-system-dependencies/service-maps/introduction-service-maps/).\n\n## More info\n\nCheck out the [documentation](https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs/) to learn more about New Relic monitoring for Hapi. **EOL notice:** We're discontinuing support for several capabilities in November 2021, including Hapi versions prior to Hapi 19.2. For more details, including how you can easily prepare for this transition, see our [Explorers Hub post](https://discuss.newrelic.com/t/important-upcoming-changes-to-capabilities-and-support-across-node-agents-suggested-pagerduty-responders-incident-workflows-and-kubernetes-instrumentation/164481?u=dholloran)."
```

Comment:
I don't think you need use the `\n\n` here for a line break. You can just put a line break in the markdown as a new line.

Reply:
That's the way the file was when I added the new text. about EOL notice. I'm not comfortable with messing around with the existing code.